src/media/css/normalize.css

```diff
@@ -0,0 +1,424 @@
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
```

Comment:
It simply seems unnecessary but a lot of work has been done with its presence already there and it might be a bad idea to cause misalignment issues at this stage - it's fine to keep it, hopefully the MIT license it OK for the project.